src/Mod/Sketcher/Gui/CommandCreateGeo.cpp

```diff
@@ -5063,11 +5063,17 @@ namespace SketcherGui {
 class DrawSketchHandlerFillet: public DrawSketchHandler
 {
 public:
-    DrawSketchHandlerFillet(int filletType) : filletType(filletType), Mode(STATUS_SEEK_First), firstCurve(0) {}
+    enum FILLET_TYPE {

```

Comment:
We usually reserve all caps for macros. I fear that would be misleading for other developers.

Could we have more descriptive names so that the reader better understands the function requested?

enum FilletType {
   NormalFillet,
   ConstraintPreservingFillet
};

Reply:
Sure! Go for it.